src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java

```diff
@@ -122,13 +122,16 @@ public void should_get_1_incomplete_envelope_from_yesterday() {
             envelope("D.zip", "Z", Status.COMPLETED)
         );
 
-        // and update createAt to yesterday
+        // and update createAt to 2h ago
         entityManager.createNativeQuery(
-            "UPDATE envelopes SET createdat = '" + now().minusDays(1) + "' WHERE zipfilename IN ('A.zip', 'B.zip')"
+            "UPDATE envelopes SET createdat = '" + now().minusHours(2) + "' WHERE zipfilename IN ('A.zip', 'B.zip')"
         ).executeUpdate();
 
+        // when
+        var incompleteCount = repo.getIncompleteEnvelopesCountBefore(now().minusHours(1));
+
         // then
-        assertThat(repo.getIncompleteEnvelopesCountBefore(now().toLocalDate())).isEqualTo(1);
+        assertThat(incompleteCount).isEqualTo(1);
```

Comment:
It may be worth changing this method to return a list of file names, not just the number of them.
Perhaps a future PR.

Reply:
definitely would be useful. container + name :shipit: